src/components/general/FilterPane/index.tsx

```diff
@@ -106,9 +108,10 @@ function FilterPane<T>({
                     <div className={styles.collapseExpandButtonContainer}>
                         <CollapseExpandButton onClick={toggleCollapsed} />
                     </div>
+                    {!isCollapsed && headerComponent}
```

Comment:
This should perhaps be wrapped in a div with `flex-grow: 1;` or something and ensure that it's contained within the header

Reply:
I have set the header to "push" the collapseButton and headercomponent appart. 
setting headerComponet to grow, breaks this abit. 
Would need some changes to justify the content internally then and checking if its left or right, to justify it to start / end. 
But I dont think it does much either way when testing it. 

As it stands now. 
If the component is to wide. It will just disappear outside, as overflow is hidden. 
If its to tall it will grow the header in height.